src/macaron/repo_finder/provenance_extractor.py

```diff
@@ -277,16 +271,6 @@ def check_if_input_repo_commit_provenance_conflict(
         )
         return True
 
-    # Check the provenance commit against the input commit.
-    if digest_input and provenance_commit_digest and digest_input != provenance_commit_digest:
-        logger.debug(
-            "The commit digest from input does not match what exists in the provenance. "
-            "Input Commit: %s, Provenance Commit: %s.",
-            digest_input,
-            provenance_commit_digest,
-        )
-        return True
-
```

Comment:
I think the reason we remove this validation is because if the user provide a commit that doesn't map to any  tag, then the provenance finder couldn't find any provenance for the analysis too. 
And even when the user provide a provenance as input, and also provide a commit as input, the [provenance_commit_check](https://github.com/oracle/macaron/blob/9c4444586afce02ca00fe62bae76aed9388684a1/src/macaron/slsa_analyzer/checks/provenance_commit_check.py) would be able to catch that inconsistency.
Is this correct?

Reply:
That's right.